mempool/src/tests/shared_mempool_test.rs

```diff
@@ -779,166 +779,6 @@ fn test_vfn_multi_network() {
     smp.assert_no_message_sent(&vfn_0);
 }
 
-#[test]
-fn test_fn_failover() {
-    // Test vfn failing over to fallback network
```

Comment:
🥳 
we should still validate that if the upstream is down that the txn is committed, do we have a test for that?

Reply:
Yeah, working on replacing this current test with a new one, that tests with the other failover mode.

There is a different test that tests the "everything is running" case already.